events/EiffelActivityQueuedEvent/simple.json

```diff
@@ -0,0 +1,35 @@
+{
+  "meta": {
+    "domainId": "eiffel000.seli.fem10",
+    "id": "acqueued-ec41-488e-93f2-fc2702b32f21",
+    "type": "EiffelActivityQueuedEvent",
+    "version": "4.0",
+    "time": 1234567890,
+    "tags": [
+      "unit-test"
+    ],
+    "source": {
+      "host": "eiffel000.seli.fem101",
+      "pid": 6132,
+      "name": "qwerty",
+      "serializer": {
+        "groupId": ""
+      },
+      "uri": "http://myhost:8080/additional/information/here"
+    }
+  },
+  "data": {
+    "name": "e.g. Component X Build",
+    "category": "e.g. Component Build",
+    "trigger": {
+      "type": "MESSAGE"
+    },
+    "description": "Triggered by something ",
```

Comment:
I think we should try to make examples that could be real. Suggest "Triggered by something " is replaced by "Triggered by EiffelArtifactPublishedEvent"